src/build/__tests__/build-test.js

```diff
@@ -9,15 +9,30 @@ import {
   GraphQLBoolean,
   GraphQLID,
 } from 'graphql/type';
+import { DirectiveLocation, GraphQLDirective } from 'graphql/type/directives';
+
 import { parse } from 'graphql/language';
 import { expectTypesEqual, expectSchemasEqual } from './comparators';
 import build, { buildTypes } from '../';
 
+const customDirective = new GraphQLDirective({
+  name: 'Dir1',
+  description:
+  'My custom uppercase',
+  locations: [
+    DirectiveLocation.FIELD,
+  ],
+});
```

Comment:
A couple of things here (very nitpicky 😅 ).

- For consistency, can we rename the variable `CustomDirective` (pascal case) and change the name field to `'CustomDirective'`?
- For style consistency, can the `description` field value be on the same line instead of a new line?
- Again for consistency, can lines 22 - 24 be condensed into a single line?
- Finally, can the directive description be a little be more descriptive? (e.g., 'Custom directive') You could also simply remove the entire description (probably prefer this as it's unnecessary for the test).